src/confluent_kafka/__init__.py

```diff
@@ -19,7 +19,11 @@
 from .deserializing_consumer import DeserializingConsumer
 from .serializing_producer import SerializingProducer
 from .error import KafkaException, KafkaError
-from ._model import Node, ConsumerGroupTopicPartitions, ConsumerGroupState
+from ._model import (Node,  # noqa: F401
+                     ConsumerGroupTopicPartitions, 
+                     ConsumerGroupState, 
+                     TopicCollection, 
```

Comment:
Add `TopicCollection` and `TopicPartitionInfo` to index.rst in the `Supporting classes` section

Reply:
At the end, check if the doc renders correctly